hugegraph-hbase/src/main/java/org/apache/hugegraph/backend/store/hbase/HbaseSessions.java

```diff
@@ -418,7 +419,11 @@ default R scan(String table, Set<byte[]> prefixes) {
          */
         default R scan(String table, byte[] startRow, boolean inclusiveStart,
                        byte[] prefix) {
+            short value = (short) (((startRow[0] << 8) | (startRow[1] & 0xFF)) +1);
+            byte[] endRow = ByteBuffer.allocate(2).putShort(value).array();
```

Comment:
not quiet clear with it，could u explain it more？

and for any of the scan query condition，we always add the end-row？

could u give a case for it（before vs after），and check the ci problems